homeassistant/components/services.yaml

```diff
@@ -320,6 +320,17 @@ axis:
         description: What parameter to operate on. [Required]
         example: 'package=VideoMotionDetection'
 
+deconz:
```

Comment:
We should no longer add more things to this file. Instead rename your component to `deconz/__init__.py` and add a `deconz/services.yaml`

Reply:
Changed